src/events/tab_event_handler.ts

```diff
@@ -64,6 +68,9 @@ export class tabEventHandler {
 		if (tabs.length === 0) return;
 		// Adding to buffer
 		this.addToBuffer(repoId, createdAt, tabs);
+	} catch(error) {
+		CodeSyncLogger.error(`[handleTabChangeEvent] Error processing tabs, error: ${error}`);
```

Comment:
@saadberry Kindly see exiting usage. 
e.g. 
![image](https://github.com/user-attachments/assets/6ef319d7-d2c0-4aaf-b443-92df2175006e)

Reply:
Thanks for giving me reference of existing implementation.

I have updated as:

![image](https://github.com/user-attachments/assets/240e0648-0ecf-4036-979d-0034162caf57)

Reply:
commit: https://github.com/codesyncapp/codesync-vs/pull/238/commits/aaecf12fee9149d2625d974c66f0721c371ce08c